AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.kt

```diff
@@ -297,6 +297,14 @@ open class Reviewer : AbstractFlashcardViewer() {
         val did = extras.getLong("deckId", Long.MIN_VALUE)
         Timber.d("selectDeckFromExtra() with deckId = %d", did)
 
+        // deckId does not exist, load default (#12910)
+        // TODO delete deck shortcut if the deck does not exist anymore
+        // TODO don't start reviewing the default deck if a shortcut for a deleted deck is launched
+        if (col.decks.get(did, _default = false) == null) {
+            Timber.d("selectDeckFromExtra() deckId '%d' doesn't exist", did)
```

Comment:
I suggest this is actually much higher - it should be a warn level log. My rule of thumb for logs is:
- error: this is something terrible, we probably cannot recover and for data safety should exit the app / log a report in our crash system
- warn: this is something terrible, but we can recover and operate normally. Log above info at least so that if it does crash later it shows up in our crash report system's logcat extract
- info: this is interesting enough that if there is a crash we want to see it in crash system logcat report
- debug: only useful for developers
- verbose: only useful for developers inspecting an area deeply

A crash would be error, this avoids a crash but is a serious problem (for the user at least) and we recover, so warn

```suggestion
            Timber.w("selectDeckFromExtra() deckId '%d' doesn't exist", did)
```